crates/ruff_workspace/src/options.rs

```diff
@@ -241,13 +241,13 @@ pub struct Options {
     /// included here not for configuration but because we lint whether e.g. the
     /// `[project]` matches the schema.
     ///
-    /// If [preview](https://docs.astral.sh/ruff/preview/) is enabled, the default
-    /// includes notebook files (`.ipynb` extension). You can exclude them by adding
-    /// `*.ipynb` to [`extend-exclude`](#extend-exclude).
+    /// Notebook files (`.ipynb` extension) are included by default on Ruff 0.6.0+.
+    /// On older versions, they are included by default only if
+    /// [preview](https://docs.astral.sh/ruff/preview/) mode is enabled.
```

Comment:
This is only true for one version. Seems a bit of a stretch :) I would probably remove the section because we normally only document the current behavior.

Reply:
makes sense